R/pkg/R/deserialize.R

```diff
@@ -237,7 +237,10 @@ readDeserializeInArrow <- function(inputCon) {
   if (requireNamespace1("arrow", quietly = TRUE)) {
     RecordBatchStreamReader <- get(
       "RecordBatchStreamReader", envir = asNamespace("arrow"), inherits = FALSE)
-    as_tibble <- get("as_tibble", envir = asNamespace("arrow"))
+    # Arrow drops `as_tibble` since 0.14.0, see ARROW-5190.
+    # nolint start
+    as_tibble <- get0("as_tibble", envir = asNamespace("arrow"), ifnotfound = NULL)
```

Comment:
hm, `get0` is under `base`. I wonder why it fails CRAN check as well .. Can we maybe just try `exists("as_tibble", envir=asNamespace("arrow"))` instead?

Reply:
Yeah, in DataFrame.R it passed the test...

Anyway let me change to `exists` and see if it is ok.